pkg/aggregator/demultiplexer.go

```diff
@@ -684,6 +673,67 @@ func (d *AgentDemultiplexer) GetMetricSamplePool() *metrics.MetricSamplePool {
 	return d.statsd.metricSamplePool
 }
 
+// GetRoutinesDistribution returns how many routines should be spawned for the
+// DogStatsD pipeline and how many DogStatsD workers should be running.
+func GetRoutinesDistribution() (int, int) {
```

Comment:
I think the method name could be better. I need the docstring to understand what it's doing.

WDYT about ?
```
GetDogstatsdWorkerAndPipelineCount()
```

Also it seems we never use both returned value (we use either the number of workers or the number of pipelines). Should we split this function in two ?

Reply:
The pipeline count is dependent of the worker count so even if only one is used I'll prefer to stick with this. But I've renamed using your suggestion 👍